_posts/en/newsletters/2019-08-14-newsletter.md

```diff
@@ -0,0 +1,89 @@
+---
+title: 'Bitcoin Optech Newsletter #59'
+permalink: /en/newsletters/2019/08/14/
+name: 2019-08-14-newsletter
+slug: 2019-08-14-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter briefly describes two discussions on the
+Bitcoin-Dev mailing list, one about Bitcoin vaults and one about
+reducing the size of public keys used in taproot.  Also included are our
+regular sections about bech32 sending support and notable changes in
+popular Bitcoin infrastructure projects.
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+
+## Action items
+
+- **Optech [San Francisco][workshop-sf] (Sep 24) and [New York
+  City][workshop-nyc] (Sep 27) workshops:**  member companies should [send us an
+  email][optech email] to reserve spots for your engineers. Planned topics
+  include presentations and coding demos involving schnorr, MuSig, taproot, and tapscript.
+
+## News
+
+- **Bitcoin vaults without covenants:** In a [post][vault1] to the
+  Bitcoin-Dev mailing list, Bryan Bishop describes how pre-signed
+  transactions paid to scripts using `OP_CHECKSEQUENCEVERIFY` (CSV)
+  could allow users to detect and block attempts to steal their money by
+  a thief who had gained access to the user's private keys, a capability
+  previously referred to as providing *Bitcoin vaults.*  After
+  describing at length both the basic protocol and several possible
+  variations, Bishop made a second [post][vault2] describing one case
+  where it would still be possible to steal from the vaults, although he
+  also suggests a partial mitigation that would limit losses to a
+  percentage of the protected funds and he requests proposals for the
+  smallest necessary change to Bitcoin's consensus rules to fully
+  mitigate the risk.
+
+- **Proposed change to schnorr pubkeys:** Pieter Wuille started a
+  [thread][pubkey32] on the Bitcoin-Dev mailing list requesting feedback
+  on a proposal to switch [bip-schnorr][] and [bip-taproot][] to using
+  32-byte public keys instead of the 33-byte [compressed pubkeys][]
+  previously proposed.  Compressed pubkeys include a bit to indicate to
+  verifiers whether their Y coordinate is part of the top half ("odd")
```

Comment:
I suspect you're correct.  I suspect where the linked description of compressed pubkey (written by me) went wrong is using a curve over natural numbers as an example---in that case, there's no even/odd distinction because `y'` would just be negative `y`.  In any case, dropping the extra details simplifies the description, so I've done that.

Reply:
Right - over the natural numbers (ie not over a finite field of odd order), then `y` and `y'` would both be even or both be odd.